collections/ansible_collections/demo/satellite/roles/register_host/tasks/main.yml

```diff
@@ -51,3 +54,51 @@
   rhsm_repository:
     name: "{{ rhsm_enabled_repos }}"
     state: enabled
+
+#These should be included in the image
+- name: Install cockpit
+  yum:
+    name:
+      - cockpit
+      - cockpit-ws
+      - cockpit-bridge
+      - cockpit-composer
+      - cockpit-dashboard
+      - cockpit-doc
+      - cockpit-packagekit
+      - cockpit-pcp
+      - cockpit-system
+      - subscription-manager-cockpit
+    state: latest
+
+- name: Install RHEL7 specific packages
+  yum:
+    name:
+      - cockpit-docker
+      - cockpit-leapp
+    state: latest
+  when: ansible_distribution_major_version=="7"
+
+- name: Install RHEL8 specific packages
+  yum:
+    name:
+      - cockpit-session-recording
+      - cockpit-podman
+    state: latest
+  when: ansible_distribution_major_version=="8"
+
+# Not these packages
+- name: install satellite client
+  yum:
+    name:
+      - katello-host-tools
+      - katello-host-tools-tracer
+    state: latest
+    
+- name: enable remote execution
+  authorized_key:
+    user: root
```

Comment:
I don't think we should be encouraging root ssh as a best practice. could this key be added to "{{ ansible_user }}"?

Reply:
I can make that change